evaluation/red_teaming.py

```diff
@@ -27,48 +30,70 @@ async def run_red_teaming(
     config: dict,
     red_teaming_llm: PromptChatTarget,
     prompt_target: PromptChatTarget,
+    max_turns: int,
+    compare: bool,
```

Comment:
Let's not have this boolean here. We should only have one function to run the red teaming either for one or multiple models.